CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [1.0.0-beta.2] - Unreleased
 ### Fixed
 - Hide login image column on small screen devices.
+### Changed
+- Parse device Attributes field in place of removed Metadata field.
```

Comment:
I don't like it, this is uninteresting for users. "- Device metadata are called device attributes now. Astarte >= 1.0.0-beta.2 is required."